MetaDataset.py

```diff
@@ -195,6 +195,10 @@ def _load_seq_list(self, seq_list_file=None):
       if seq_list_file.endswith(".pkl"):
         import pickle
         seq_list = pickle.load(open(seq_list_file, 'rb'))
+      elif seq_list_file.endswith(".gz"):
+        import gzip
+        # needs strip, as gzip returns sequences with \n at end
```

Comment:
`readlines` always returns with `"\n"`, also for normal files.
You could do `read().splitlines()` instead, which will not have that `"\n"`. Not sure which is faster. Anyway, the comment is slightly confusing.